cloudbuild.yaml

```
@@ -149,6 +149,14 @@ steps:
     bazel run --host_force_python=PY2 //experimental/dotnet:debug_debian9
     bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_debian10
     bazel run --host_force_python=PY2 //experimental/dotnet:debug_debian10
+
+    bazel run --host_force_python=PY2 //experimental/dotnet_core_aspnet:dotnet_core_aspnet
```

Let's explicitly spell out `_debian9` to eliminate any possible ambiguity.
```suggestion
    bazel run --host_force_python=PY2 //experimental/dotnet_core_aspnet:dotnet_core_aspnet_debian9
```